app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[new create]
+
+  def new
```

Comment:
I think we can remove new if we're putting the "new reservation" form on the unicorn's page